runtime-testsuite/src/org/antlr/v4/testgen/TestGenerator.java

```diff
@@ -266,11 +380,20 @@ private void importLanguageTemplates(STGroup testGroup, STGroup languageGroup) {
 		}
 	}
 
+    public String render(ST template) {
```

Comment:
📝 This method seems unnecessary since the line endings for the affected files is being managed by Git itself.